Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -934,7 +935,8 @@ private final class TokenStreamCreator: SyntaxVisitor {
 
   func visit(_ node: CodeBlockItemSyntax) -> SyntaxVisitorContinueKind {
     before(node.firstToken, tokens: .open)
-    after(node.lastToken, tokens: .close, .break(.reset, size: 0))
+    let resetSize = node.semicolon != nil ? 1 : 0
```

Comment:
Let me know if this calculation of reset size should be a common helper function to be called by `CodeBlockItemSyntax` and `MemberDeclListItemSyntax`

Reply:
I don't think that's necessary; at this point I don't foresee us wanting to generalize the number of spaces to anything else, so factoring out doesn't buy us much.

It's interesting to observe that these two methods are identical, so we _could_ go as far as factor them out in their entirety into a single helper function, but I don't think we want to do that either. It's conceivable that at some point in the future we'd want to handle multi-statement lines differently than multi-decl lines, and combining the logic would make that more difficult. So I think it's all fine the way it is.

Reply:
Agreed